patches/third_party-blink-renderer-core-workers-shared_worker_content_settings_proxy.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/core/workers/shared_worker_content_settings_proxy.h b/third_party/blink/renderer/core/workers/shared_worker_content_settings_proxy.h
+index 95a3b6e30bc39443ae60683fb5ad613bb41fca6f..14d8c0e006e453cb3526a81c02e1e3d48c6f67c0 100644
+--- a/third_party/blink/renderer/core/workers/shared_worker_content_settings_proxy.h
++++ b/third_party/blink/renderer/core/workers/shared_worker_content_settings_proxy.h
+@@ -26,6 +26,7 @@ class SharedWorkerContentSettingsProxy : public WebContentSettingsClient {
+   // WebContentSettingsClient overrides.
+   bool AllowStorageAccessSync(StorageType storage_type) override;
+ 
++  BRAVE_SHARED_WORKER_CONTENT_SETTINGS_PROXY_H
```

Comment:
I think this can also be replaced with `#define AllowStorageAccessSync`?
```
#define AllowStorageAccessSync                             \
  AllowFingerprinting(bool enabled_per_settings) override; \
  BraveFarblingLevel GetBraveFarblingLevel() override;     \
  bool AllowStorageAccessSync(StorageType storage_type)
```

Reply:
fixed in https://github.com/brave/brave-core/commit/eb9411b9f97716dbfb49a6834b3499f3beb36129